src/NUnitTestAdapter/NUnit3TestExecutor.cs

```diff
@@ -266,7 +266,13 @@ private void RunAssembly(string assemblyName, TestFilter filter)
                     }
                 }
                 else
-                    TestLog.Info("NUnit failed to load " + assemblyName);
+                {
+                    var msgNode = loadResult.SelectSingleNode("properties/property[@name='_SKIPREASON']");
+                    if (msgNode != null && (new[] { "contains no tests", "Has no TestFixtures" }).Any(msgNode.GetAttribute("value").Contains))
+                        TestLog.Info("Assembly " + assemblyName + " contains no tests supported by this runner: " + _activeRunner.GetType().AssemblyQualifiedName);
```

Comment:
Okay, so the message should be along the lines of "NUnit3 was able to discover some tests in {assembly}, but could not run thoose tests. Please inspect your projects assembly dependencies. (TestRunner was: {AssemblyQualifiedName})" ?